packages/core/contracts/data-verification-mechanism/implementation/Staker.sol

```diff
@@ -159,26 +159,27 @@ abstract contract Staker is StakerInterface, Ownable, Lockable, MultiCaller {
         cumulativeStake -= amount;
         voterStake.pendingUnstake = amount;
         voterStake.stake -= amount;
-        voterStake.unstakeRequestTime = SafeCast.toUint64(getCurrentTime());
+        voterStake.unstakeTime = SafeCast.toUint64(getCurrentTime()) + unstakeCoolDown;
```

Comment:
there was another comment elsewhere in the code about this type casting being redundant. I think the same would apply here (it can just be an unsafe `uint64` cast).